force-app/main/default/triggers/opportunityTrigger.trigger

```diff
@@ -0,0 +1,51 @@
+trigger opportunityTrigger on Opportunity (before insert, before update, after update, before delete) {
+ // solution to question 5
+    if(Trigger.isUpdate && trigger.isBefore){
+        for(Opportunity opp : Trigger.new){
+            if(Opp.Amount < 5000) {
+                opp.addError('Opportunity amount must be greater than 5000');
+            } 
+        }
+    }
+    // solution to question 6 - banking industry logic isn't working
+    if(Trigger.isBefore && Trigger.isDelete){
+        // This map will hold the Opportunity ID and related Accounts.
+        Map<Id,Account> MapOppIDwithAccounts = new map<Id, Account>();
+        
+        // This map obtains Opportunities where the parent Account industy is banking and where opportunity stage name is closed won.
+        Map<Id, Opportunity> MapOpps = new Map<Id,Opportunity>([SELECT Name, ID, AccountID FROM Opportunity WHERE StageName = 'Closed Won' AND Account.Industry = 'Banking']);
+        System.debug(MapOpps);
+        for(Opportunity opps : Trigger.old ) {  //MapOpps.values()
+            opps.addError('Cannot delete closed opportunity for a banking account that is won');
+        }
+    }
```

Comment:
You will need a query to get the related Accounts.  Related records are not available in Trigger context so you must query them.